src/components/signup/components/Signup.js

```diff
@@ -45,7 +53,8 @@ class Signup extends React.Component {
                 typeOfProducts:values.typeOfProducts,
                 username: values.username,
             };
-            signupRequest(user);
+            signupRequest(payload);
```

Comment:
you dont have to rename `user` here to payload, you can call is `user`. It is on the actions side that we will map it `user` to `payload`